### ISSUE TITLE
Extract Face region , timestamp of each unique face appearance , active speaker or not from a video , in Json or any format .

### ISSUE THREAD

**Issue by 808Code** [TaoRuijie/TalkNet_ASD#65]
Using the project how would I begin to extract  each unique face in a video , and based on that face its timestamp from when it occurs and the region , as well as is it  active speaker or not  ? ; in some sort of json or any format , rather than all these things directly being output into video form in the  demo folder.

Thankyou.

**Comment by RishabhMathur06** [comment]
Yes please, if that can be done. please suggest a way to do that using this model.
